docs/conf.py

```diff
@@ -0,0 +1,13 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Copyright block is missing in all the new files. Please add as appropriate

Reply:
Addressed in [add copyright block to code files](https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/pull/145/commits/8d9ec074a83fe9868dc52e7ee2ccf0a5d18e0cde)